src/index.js

```diff
@@ -372,4 +384,31 @@ Dropzone.defaultProps = {
   minSize: 0
 };
 
+Dropzone.propTypes = {
```

Comment:
Hmm, can't see this change for the whole PR but only for the last commit. So, I guess, nevermind my comment.